pyproject.toml

```diff
@@ -33,7 +33,7 @@ dependencies = [
     'pooch',
     'scooby>=0.5.1',
     "typing-extensions; python_version < '3.8'",
-    'vtk',
+    'vtk>=9.0.0',
```

Comment:
We don't want to add any version constraints on VTK as the VTK version string can be set to all kinds of different things depending on where it came from and we do not want pip to mess with an already installed VTK

Reply:
good point (https://github.com/conda-forge/vtk-feedstock/issues/234). I'll leave a comment instead.